### PR TITLE
[occm] add priorityClassName to helm chart

### PR DESCRIPTION
**Pull request by iceman2077**
**What this PR does / why we need it**:
ability to add priority class for occm

**Special notes for reviewers**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dulek** [comment]
/ok-to-test

I think this makes sense.

File: charts/openstack-cloud-controller-manager/templates/daemonset.yaml [hunk 1]
```diff
@@ -95,6 +95,9 @@ spec:
       initContainers: {{ toYaml .Values.extraInitContainers | nindent 6 }}
       {{- end }}
       hostNetwork: true
+      {{- if .Values.priorityClassName }}
```
Comment (zetaab) [inline charts/openstack-cloud-controller-manager/templates/daemonset.yaml:98]: you need to increase the chart version in https://github.com/kubernetes/cloud-provider-openstack/blob/master/charts/openstack-cloud-controller-manager/Chart.yaml#L7 use like alpha.2

**Comment by dulek** [comment]
/lgtm

@iceman2077: I think you got to do CLA: https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by iceman2077** [comment]
> /lgtm
> 
> @iceman2077: I think you got to do CLA: https://github.com/kubernetes/community/blob/master/CLA.md

looks like some of the commits have no linked email,
will update and create new pr

**Pull request closed without merging**